### PR TITLE
populating data to columns

### PR DESCRIPTION
**Pull request by kjain110**
https://issues.folio.org/browse/MODLISTS-38

### CODE REVIEW

**Review by bvsharp — COMMENTED** [review]
Since 1.0.0 is already released, I think you'll need to create a new version of this file with the changes in the v1.0.1 directory

File: src/main/resources/db/changelog/changes/v1.0.1/update-list-details-table.sql
Comment (mweaver-ebsco) [inline src/main/resources/db/changelog/changes/v1.0.1/update-list-details-table.sql:1]: If this is updating the existing lists, then this should probably be a pair of `UPDATE` statements, right?

Comment (kjain110) [reply src/main/resources/db/changelog/changes/v1.0.1/update-list-details-table.sql:1]: We are appending the values of one column and that's why we are using this way.

Comment (kjain110) [reply src/main/resources/db/changelog/changes/v1.0.1/update-list-details-table.sql:1]: @mweaver-ebsco done

File: src/main/resources/db/changelog/changes/v1.0.1/sql/update-list-details-table.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+UPDATE list_details SET fields =
+  '{user_id, user_first_name, user_last_name, user_full_name, user_active, user_barcode, user_expiration_date, user_patron_group_id, user_patron_group, id, loan_status, loan_checkout_date, loan_due_date, loan_policy_id, loan_policy_name, loan_checkout_servicepoint_id, loan_checkout_servicepoint_name, item_holdingsrecord_id, instance_id, instance_title, instance_primary_contributor, item_id, item_barcode, item_status, item_material_type_id, item_material_type, holdings_id, loan_checkin_servicepoint_id, loan_checkin_servicepoint_name, loan_return_date, item_call_number}'
```
Comment (bvsharp) [inline src/main/resources/db/changelog/changes/v1.0.1/sql/update-list-details-table.sql:2]: I think item_holdingsrecord_id can be removed from this. We already have holdings_id and item_holdingsrecord_id isn't a field in the drv_loan_details view.

**Review by bvsharp — APPROVED** [review]

**Pull request merged**